package.json

```diff
@@ -27,14 +27,17 @@
     "sodium-universal": "^3.0.4"
   },
   "devDependencies": {
+    "@digidem/types": "^2.0.0",
     "@types/crc": "^3.4.0",
     "@types/lodash": "^4.14.178",
     "@types/node": "^17.0.0",
     "@types/sodium-native": "^2.3.5",
     "@types/tap": "^15.0.5",
     "@types/tape": "^4.13.2",
     "documentation": "^13.2.5",
+    "hypercore": "^10.18.5",
     "prettier": "^2.5.1",
+    "random-access-memory": "^6.2.0",
     "tap": "^15.1.5",
     "typescript": "^4.5.4"
```

Comment:
need to upgrade Typescript to >5.1 in order for the digidem types declarations to work properly